pkg/server/events_test.go

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Use a fake clock for the unit test.
https://github.com/containerd/cri/tree/master/vendor/k8s.io/apimachinery/pkg/util/clock